mrp_production_grouped_by_product/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============================
+Production Grouped By Product
+=============================
+
+Groups pending productions by product.
```

Comment:
Better text:
```
When you have several sales orders with make to (MTO) order products that require to be manufactured, you end up with one manufacturing order for each of these sales orders, which is very bad for the management. With this module, each time an MTO manufacturing order is required to be created, it first checks that there's no other existing order not yet started for the same product and bill of materials, and if there's one, then the quantity of that order is increased instead of creating a new one.
```